### PR TITLE
More details about fees and core pools redirection

### PR DESCRIPTION
**Pull request by Tritium-VLK**

### CODE REVIEW

File: docs/concepts/governance/multisig.md [hunk 1]
```diff
@@ -7,19 +7,18 @@ The core of Balancer smart contracts are immutable and do not use proxies or oth
...
+| LM Multisig            | Used to manage gauges and Reward Tokens and manage liquidity supplied to multichain (bridge). New Gauge requests go here.               | [MAINNET](https://gnosis-safe.io/app/eth:0xc38c5f97B34E175FFd35407fc91a937300E33860/home), [ARBI](https://gnosis-safe.io/app/arb1:0xc38c5f97B34E175FFd35407fc91a937300E33860/home), [POLYGON](https://gnosis-safe.io/app/matic:0xc38c5f97B34E175FFd35407fc91a937300E33860/home) | `0xc38c5f97B34E175FFd35407fc91a937300E33860` | [BAL Maxis](#operational-multisigs-signer-set-aka-balancer-maxis) + [Markus Koch - blabs](https://etherscan.io/address/0x6bB4720473d4D7133f944785e5EE1A650C07f34e) |
```
Comment (Tritium-VLK) [inline docs/concepts/governance/multisig.md]: @markusbkoch Your name is being connected to this signer address on the LM multisig here.  Just want to make sure to get sign-off from you before pushing that to the docs pages.

File: docs/concepts/governance/bal-token.md [hunk 2]
```diff
@@ -21,11 +21,11 @@ Balancer Governance Token \(BAL\) is the core token behind the Balancer protocol
...
+At the start of BAL token incentives 145,000 BAL were minted every week. In Q1 2022, [veBAL was introduced](https://forum.balancer.fi/t/introducing-vebal-tokenomics/2512) to replace fixed weekly emissions. veBAL includes an annual reduction in emissions rate such that the program should run until around 2050, leaving a total supply of around 94/100 million tokens upon completion and assuming no other governance is enacted to mint part of the remaining unallocated supply.
 
 TOTAL Tokens to be emitted by veBAL: 47,520,700
 
-The current emissions rate can be understood by looking at the [Balancer Token Admin Contract](https://etherscan.io/address/0xf302f9f50958c5593770fdf4d4812309ff77414f#readContract)
+The current emissions rate can be understood by looking at the Balancer Token Admin Contract
```
Comment (mikemcdonald) [inline docs/concepts/governance/bal-token.md]: Any reason why this link is removed?

File: docs/concepts/governance/bal-token.md [hunk 3]
```diff
@@ -150,12 +149,9 @@ This results in the following emissions schedule:
     }
   }
 }
-```
-
 :::
 
-## Distribution
-
+Distribution
```
Comment (mikemcdonald) [inline docs/concepts/governance/bal-token.md]: This should probably keep a header format

File: docs/concepts/governance/bal-token.md [hunk 4]
```diff
@@ -200,32 +195,29 @@ This chart outlines the allocation amounts. More details below.
...
 
 25M tokens were allocated to founders, stock options, advisors, and investors, all subject to vesting periods.
 
 #### Ecosystem Fund
 
-5M were allocated for the Ecosystem Fund. This fund will be deployed to attract and incentivize strategic partners who will help the Balancer ecosystem grow and thrive. BAL holders will ultimately decide how this fund is used over the coming years.
+5M were allocated for the Ecosystem Fund. This fund will be deployed to attract and incentivize strategic partners who will help the Balancer ecosystem grow and thrive. veBAL holders will ultimately decide how this fund is used over the coming years.
 
-Note that as part of the veBAL conversion, the entire ecosystem fund was transferred into the [DAO multisig](https://etherscan.io/address/0x10A19e7eE7d7F8a52822f6817de8ea18204F2e4f)
+Note that as part of the veBAL conversion, the entire ecosystem fund was transferred into the [DAO multisig](https://polygonscan.io/address/0xeE071f4B516F69a1603dA393CdE8e76C40E5Be85).
```
Comment (mikemcdonald) [inline docs/concepts/governance/bal-token.md]: This link is broken. Isn't the original correct?

File: docs/concepts/governance/multisig.md [hunk 5]
```diff
@@ -31,32 +30,13 @@ But getting to that ideal, long-term vision of a truly decentralized and effecti
...
 
 Balancer V2 contracts, on the other hand, do allow for some tweaking of core protocol parameters, for instance, in turning on protocol fees. As a placeholder for a future on-chain DAO, such limited admin powers has been initially granted to a Multisig.
 
-Balancer strives to continue to automate operations and governance execution. While the eventual goal is still to move the entire governance and execution on-chain, the current Multisigs have proven themselves as reliable executors of the wishes of veBAL voters. In the long run, BAL holders are expected to retire the Multisig in favor of a full-fledged Balancer DAO.
+Balancer strives to continue to automate operations and governance execution. While the eventual goal is still to move the entire governance and execution on-chain, the current Multisigs have proven themselves as reliable executors of the wishes of veBAL voters. In the long run, BAL holders are expected to retire the Multisigners in favor of a full-fledged Balancer DAO.
```
Comment (mikemcdonald) [inline docs/concepts/governance/multisig.md]: The original `Multisig` seems fine to me

**Review by mikemcdonald — CHANGES_REQUESTED** [review]

File: docs/concepts/governance/bal-token.md (hunk 3 above)
Comment (Tritium-VLK) [reply docs/concepts/governance/bal-token.md]: Correct.  Pushed changes.

File: docs/concepts/governance/bal-token.md (hunk 4 above)
Comment (Tritium-VLK) [reply docs/concepts/governance/bal-token.md]: Yeah it is.  Was merging in some changes from others here.

**Comment by Tritium-VLK** [comment]
@mikemcdonald thanks for catching all that.  Won't merge a bunch of other peoples ideas together like that again. Showing @0xdanko how to use github.

**Comment by mikemcdonald** [comment]
Address for polygon ecosystem fund is still pointing to mainnet. Once that's fixed I'll get this merged

**Review by mikemcdonald — APPROVED** [review]

**Pull request merged**